forest/src/daemon.rs

```diff
@@ -75,9 +75,29 @@ pub(super) async fn start(config: Config) {
     });
 
     let db_rpc = Arc::clone(&db);
-    let rpc_thread = task::spawn(async {
-        start_rpc(db_rpc).await;
-    });
+
+    // Use the provided RPC port and listener or use the default one
+    let rpc_port = config
+        .rpc_port
+        .unwrap_or_else(|| "/api".to_string())
+        .to_owned();
+    let rpc_listen = config
+        .rpc_listen
+        .unwrap_or_else(|| "127.0.0.1:8080".to_string())
+        .to_owned();
```

Comment:
do this within the if statement (shouldn't have to do this if rpc disabled). When the previous comment comes in this can be removed entirely anyway